charts/airflow/templates/config/secret-config.yaml

```diff
@@ -72,7 +72,7 @@ stringData:
   ## a bash command which echos the URL encoded value of $REDIS_PASSWORD
   ## NOTE: if $REDIS_PASSWORD is non-empty, prints `:${REDIS_PASSWORD}@`, else ``
   REDIS_PASSWORD_CMD: |-
-    echo ${REDIS_PASSWORD} | python3 -c "import urllib.parse; encoded_pass = urllib.parse.quote(input()); print(f\":{encoded_pass}@\") if len(encoded_pass) > 0 else None"
+    echo "${REDIS_PASSWORD}" | python3 -c "import urllib.parse; encoded_pass = urllib.parse.quote(input()); print(f\":{encoded_pass}@\") if len(encoded_pass) > 0 else None"
```

Comment:
Actually it would be even safer to use directly is.environ instead of echo and input

Reply:
Its probably fine for now, but can you raise an issue to just use straight python, and put it in enhancements todo (see the kanban I made https://github.com/airflow-helm/charts/projects/2)

Reply:
Wait, it might not work with " now, I might have to fix it, lol

Reply:
@gsemet I raised an issue to track this enhancement idea https://github.com/airflow-helm/charts/issues/151